kinode/src/state.rs

```diff
@@ -19,7 +19,7 @@ pub async fn load_state(
     keypair: Arc<signature::Ed25519KeyPair>,
     home_directory_path: String,
     runtime_extensions: Vec<(ProcessId, MessageSender, bool)>,
-) -> Result<(ProcessMap, DB), StateError> {
+) -> Result<(ProcessMap, DB, ReverseCapIndex), StateError> {
```

Comment:
see above: no reason to serialize this data structure and persist!

Reply:
I'm not! Am I missing something?